docs/source/faq.md

```diff
@@ -115,6 +115,44 @@ It allows customization of the various parameters. An environment variable
 has precedence over the value defined in `ucx.conf`.
 The file can be created using `ucx_info -Cf`.
 
+#### Build user application with UCX
+
+UCX supports metainformation subsystem based on pkg-config tool. In order to
+build the application with UCX development libraries.  For example, this is how
+pkg-config can be incorporated in a Makefile-based build:
+```
+program: program.c
+        cc program.c $(pkg-config --cflags --libs ucx)
+```
+When linking with static UCX libraries, need to list explicitly all required
+transport modules.  For example, in order to support only cma and knem
+transports, can use this method:
+```
+program: program.c
+        cc -static program.c $(pkg-config --cflags --libs --static ucx-cma ucx-knem ucx)
```

Comment:
make shell again?  and typically you'd use $(CC) instead of hardwired cc

Reply:
yes, fixed